application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -70,12 +70,12 @@
         #if ($xwiki.exists($userReference))
           #set ($username = "$!user.getValue('first_name') $!user.getValue('last_name')")
           #if ("$!username.trim()" == '')
-            #set ($username = $user.name)
+            #set ($username = $user.documentReference.name)
```

Comment:
There is some room for improvements in terms of variable naming. `$userDoc` would state better than `$user` that we're using a document.